app/controllers/subscriptions_controller.rb

```diff
@@ -35,6 +23,18 @@ def create
     redirect_to new_subscription_path(plan: (@plan.try(:third_party_identifier) || 'premium'))
   end
 
+  def update
+    customer = Stripe::Customer.retrieve(current_user.stripe_customer_id)
+    subscription = customer.subscriptions.retrieve(customer.subscriptions.first.id)
+    subscription.plan = "premiummob"
```

Comment:
Yeah, that's being pulled out in the next step for generic updates - however that string is the 3rd party stripe id and that subscription object is from the Stripe API so we can't quite change it - we can only wrap it